Relativity.Export.Samples.Powershell/SamplesCollection/export-folder-all-with-ds-name-in-loadfile.ps1

```diff
@@ -0,0 +1,120 @@
+. "$global:rootDir\Helpers\EndpointsClass.ps1"
+. "$global:rootDir\Helpers\WriteInformationClass.ps1"
+. "$global:rootDir\Helpers\BaseExportService.ps1"
+
+# Your workspace ID: this is where we point to the workspace where we want to export from
+[int]$workspaceId = 1022188
+
+# Export settings parameters
+# Your view ID: view will provide us with available data to export, requires folder to be visible there.
+# Your folder ID: our targetted folder. If you want to export from the workspace root, the ID is different from the workspace ID.
+# ExportSourceType: ExportFolder or ExportFolderWithSubfolders
+[int]$viewId = 1003684
```

Comment:
$viewId  -> $viewID

Reply:
In some places there are 'Id' , in another 'ID' (uppercase)

Reply:
this file is copied from other example, I simply added one line with "LoadfileNameFormat":"WithDataSourceNameAndDate",  but I get you